tee-worker/ts-tests/integration-tests/examples/direct-invocation/example.ts

```diff
@@ -109,13 +111,13 @@ export async function runExample(keyPairType: KeypairType) {
     assert.isTrue(res.status.asTrustedOperationStatus[0].isInSidechainBlock);
     assert.equal(u8aToHex(res.status.asTrustedOperationStatus[1]), getTopHash(parachainApi, setUserShieldingKeyCall));
 
-    let decodedRes = parachainApi.createType(
+    const res1 = parachainApi.createType(
```

Comment:
maybe `setUserShieldingKeyRes` ?

Reply:
I was struggling with the naming - `setUserShieldingKeyRes` just seems too long but meanwhile I can't find a better name (and short)

Reply:
Nvm let me change that...